server/routes/user.routes.ts

```diff
@@ -1,14 +1,15 @@
 import { Router } from 'express';
 import { deleteUser, getUser, getUsers, postUser, putUser } from "../controllers/user.controller";
 import { emailExist, userExist } from '../middlewares/db_validator';
+import { check } from 'express-validator';
 import validateJWT from '../middlewares/validate-jwt';
 
 const router = Router();
 
 router.get('/', getUsers);
 router.post('/', [emailExist], postUser);
 router.get('/:id', [userExist], getUser);
-router.put('/:id', [validateJWT, userExist], putUser);
+router.put('/:id', [check('email', 'Email is required').isEmail(), emailExist], putUser);
```

Comment:
I suggest you change the name of emailExists for emailNotExists, because you want to check if the email is not already registered.